### DISCUSSION TITLE
求稳定版分支

### DISCUSSION THREAD

**Discussion by ThermalEng** [Boos4721/openwrt#53] [Ideas/你的好主意]
大佬，能不能开个稳定版分支，定期合并。
感觉是3600第三方固件里信号最好，速度最快的固件了，一直在追更。
但经常刷了由于各种不稳定因素又要刷回去，不想每次刷机都跟开彩蛋一样了。
请大佬开个稳定版分支，定期测试、合并。
谢谢了！

**Comment by Boos4721** [comment]
na, need tester.

**Reply by ThermalEng** [reply]
哈  我先反馈个 最近两周的版本 都存在手机上传很弱的问题 。我的手机是vivo t1，2x2 160mhz。近距离上行速度只有200mbps，隔两堵墙信号仍然有近-60dbi的情况下，基本上就没有上行速度了。昨天晚上自行编译的版本也还是这样。

**Reply by Pacio** [reply]
> na, need tester.

ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗

**Reply by Boos4721** [reply]
> > na, need tester.
> 
> ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗

![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png)
**WORKS**

**Reply by Pacio** [reply]
> > > na, need tester.
> > 
> > 
> > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> 
> ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**

不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以

**Reply by Boos4721** [reply]
> > > > na, need tester.
> > > 
> > > 
> > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > 
> > 
> > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> 
> 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以

i know, change to wpa/wpa2.

**Reply by Pacio** [reply]
> > > > > na, need tester.
> > > > 
> > > > 
> > > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > > 
> > > 
> > > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> > 
> > 
> > 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以
> 
> i know, change to wpa/wpa2.
It's been tried,but doesn't work
只能继续使用ac模式，但是该模式下内存泄漏更严重了，什么插件都没开内存就剩65m可用

**Reply by Boos4721** [reply]
> > > > > > na, need tester.
> > > > > 
> > > > > 
> > > > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > > > 
> > > > 
> > > > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> > > 
> > > 
> > > 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以
> > 
> > 
> > i know, change to wpa/wpa2.
> > It's been tried,but doesn't work
> > 只能继续使用ac模式，但是该模式下内存泄漏更严重了，什么插件都没开内存就剩65m可用

https://github.com/Boos4721/openwrt/discussions/55 or buy 1G memory

**Reply by Pacio** [reply]
> > > > > > > na, need tester.
> > > > > > 
> > > > > > 
> > > > > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > > > > 
> > > > > 
> > > > > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> > > > 
> > > > 
> > > > 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以
> > > 
> > > 
> > > i know, change to wpa/wpa2.
> > > It's been tried,but doesn't work
> > > 只能继续使用ac模式，但是该模式下内存泄漏更严重了，什么插件都没开内存就剩65m可用
> 
> #55 or buy 1G memory

ok, i'll try stable

**Reply by Pacio** [reply]
> > > > > > > > na, need tester.
> > > > > > > 
> > > > > > > 
> > > > > > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > > > > > 
> > > > > > 
> > > > > > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> > > > > 
> > > > > 
> > > > > 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以
> > > > 
> > > > 
> > > > i know, change to wpa/wpa2.
> > > > It's been tried,but doesn't work
> > > > 只能继续使用ac模式，但是该模式下内存泄漏更严重了，什么插件都没开内存就剩65m可用
> > 
> > 
> > #55 or buy 1G memory
> 
> ok, i'll try stable

![image](https://user-images.githubusercontent.com/6836705/147575177-72531203-2ce0-48bf-96b6-960eb4899b55.png)
今天编译了一下stable分支，不知道为什么每次编译到这里就报错啊? master测试了一下仍然可以正常编译

**Reply by Boos4721** [reply]
> > > > > > > > > na, need tester.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > ax6的5g wifi 的ax模式不能正常工作，切换到ac才可以，能修一下吗
> > > > > > > 
> > > > > > > 
> > > > > > > ![iShot2021-12-27 00 54 02](https://user-images.githubusercontent.com/43845654/147415017-8511e405-a016-42a7-aa33-b4b23f932ee2.png) **WORKS**
> > > > > > 
> > > > > > 
> > > > > > 不好意思我可能没说清，5G的ax模式不是完全不能用，而是个别设备不能正常连接，比如我的iphone13就连不上，但是笔记本电脑却可以
> > > > > 
> > > > > 
> > > > > i know, change to wpa/wpa2.
> > > > > It's been tried,but doesn't work
> > > > > 只能继续使用ac模式，但是该模式下内存泄漏更严重了，什么插件都没开内存就剩65m可用
> > > 
> > > 
> > > #55 or buy 1G memory
> > 
> > 
> > ok, i'll try stable
> 
> ![image](https://user-images.githubusercontent.com/6836705/147575177-72531203-2ce0-48bf-96b6-960eb4899b55.png) 今天编译了一下stable分支，不知道为什么每次编译到这里就报错啊? master测试了一下仍然可以正常编译

fixed.
